app/views/layouts/listnav/_container_link_to.html.haml

```diff
@@ -0,0 +1,43 @@
+- return if object.nil?
```

Comment:
> You mean same as `container_summary_helper.rb`?

Yes. I haven't looked at which helper modules are available when the listnav partials are invoked... I suspect we'd need to create & include one just for this purpose.

But yeah, I'm suggesting that we create `listnav_` versions of [the generic `textual_` methods](https://github.com/manageiq/manageiq/blob/master/app/helpers/container_summary_helper.rb#L80-L140) in `container_summary_helper.rb`.